README.md

```diff
@@ -151,6 +152,76 @@ Select a new parameter file or create a new parameter file to update the associa
 
 ![Image showing the parameter file creation options (none, new, and browse)](./images/undo-change-mapping-two.png)
 
+## Linked Template Support
```

Comment:
Images/animations?

Reply:
Added animation for param quick fix. Can't figure out how to re-size images for screenshots on GitHub. I'll give it another shot tomorrow.